process/sync/metablock.go

```diff
@@ -101,13 +103,20 @@ func NewMetaBootstrap(
 	base.blockBootstrapper = &boot
 	base.getHeaderFromPool = boot.getMetaHeaderFromPool
 	base.syncStarter = &boot
+	base.requestMiniBlocks = boot.requestMiniBlocksFromHeaderWithNonceIfMissing
 
 	//there is one header topic so it is ok to save it
 	hdrResolver, err := resolversFinder.MetaChainResolver(factory.MetachainBlocksTopic)
 	if err != nil {
 		return nil, err
 	}
 
+	//sync should request the missing block body on the intrashard topic
+	miniBlocksResolver, err := resolversFinder.MetaChainResolver(factory.MiniBlocksTopic)
```

Comment:
You could remove resolverFinder from MetaBootstrap struct, is unused in MetaBootstrap class beside New... method

Reply:
done